netbox_librenms_plugin/templates/netbox_librenms_plugin/inc/_module_sync.html

```
@@ -0,0 +1,28 @@
+
+<!-- Action Buttons -->
+<div class="d-flex justify-content-between align-items-center mb-3">
+    <h2>Module Sync</h2>
+    <div class="btn-list">
+        <form method="post">
+            {% csrf_token %}
+            <input type="hidden" name="server_key" value="{{ server_info.server_key|default:'' }}">
+            {% if has_librenms_id %}
+                {% with model_name=object|meta:"model_name" %}
+                    {% if model_name == "device" %}
+                        <button hx-post="{% url 'plugins:netbox_librenms_plugin:device_module_sync' pk=object.pk %}"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)